source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -35,13 +42,22 @@ partitions = this would theoretically provide you up to: ``~896,000
 PUT/POST/DELETE TPS and 1,408,000`` GET TPS if requests are distributed evenly
 across the partitions).
 
+Configuration
+^^^^^^^^^^^^^
+
 .. code-block:: none
 
   plugin {
-    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
-    obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_index_fs = compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
```

Comment:
There should't be extra space after "aws-s3:" (shouldn't have been in the original doc either)